packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1490,6 +1490,10 @@ describe("Runtime", () => {
 				};
 			};
 
+			const runtimeOptions: IContainerRuntimeOptionsInternal = {
+				flushMode: FlushModeExperimental.Async as unknown as FlushMode,
```

Comment:
Looks like another opportunity. IContainerRuntimeOptionsInternal should just allow FlushModeExperimental.Async, yeah?
But I am hoping(?) that TypeScript doesn't allow that now.?

Reply:
Right, would be a future improvement to bring that one into the internal flushMode option. Now that it's all internal we can do that at our leisure. Not sure who is using/working on that Async mode.